cmd/notary/util.go

```diff
@@ -52,3 +53,12 @@ func feedback(t *tufCommander, payload []byte) error {
 	os.Stdout.Write(payload)
 	return nil
 }
+
+// homeExpand will expand an initial ~ to the user home directory. This is supported for
+// config files where the shell will not have expanded paths.
+func homeExpand(homeDir, path string) string {
+	if path == "" || path[1] != '~' || (len(path) > 1 && path[1] != os.PathSeparator) {
```

Comment:
Should `path[1] != '~'` be `path[0] ~= '~'`?  Since we only want to expand if the first char is `~`?  (e.g. `~/.docker`).

Also, how would we expand for something like `~cyli` or ~justincormack`?  That is not the homedir of the current user, necessarily.

Reply:
ah yes, typo, fixed now.

This has never worked with `~cyli` etc so there is no change here. If you put that on the command line your shell will expand it before `notary` sees it anyway, so it is only for use in the config file, where it would be quite odd to put files in another user's home.